libraries/AP_Scripting/AP_Scripting.cpp

```diff
@@ -316,7 +316,7 @@ void AP_Scripting::handle_mission_command(const AP_Mission::Mission_Command& cmd
             mission_data = nullptr;
         }
         if (mission_data == nullptr) {
-            GCS_SEND_TEXT(MAV_SEVERITY_INFO, "Scripting: %s", "unable to receive mission command");
+            GCS_SEND_TEXT(MAV_SEVERITY_CRITICAL, "Scripting: %s", "unable to receive mission command");
```

Comment:
This one might not deserve this.  Not in immediate chance of crashing here...

Reply:
use ERROR instead

Reply:
changed to _ERROR